### PR TITLE
Added Rocky 8.5

### PR DESCRIPTION
**Pull request by ronzle**

### CODE REVIEW

File: kvm-install-vm [hunk 1]
```diff
@@ -517,7 +526,7 @@ function storpool_exists ()
 function set_sudo_group ()
 {
     case "${DISTRO}" in
-        centos?|fedora??|*-atomic|amazon?|opensuse* )
+        centos?|fedora??|rocky??|*-atomic|amazon?|opensuse* )
```
Comment (giovtorres) [inline kvm-install-vm:529]: Should this be `rocky* instead?

File: kvm-install-vm [hunk 2]
```diff
@@ -535,7 +544,7 @@ function set_cloud_init_remove ()
         centos6 )
             CLOUDINITDISABLE="chkconfig cloud-init off"
             ;;
-        centos8|centos7|amazon?|fedora??|ubuntu*|debian*|opensuse* )
+        centos8|centos7|amazon?|fedora??|rocky??|ubuntu*|debian*|opensuse* )
```
Comment (giovtorres) [inline kvm-install-vm:547]: Same here

File: kvm-install-vm (hunk 1 above)
Comment (ronzle) [reply kvm-install-vm:529]: yeah true, i will change this

File: kvm-install-vm (hunk 2 above)
Comment (ronzle) [reply kvm-install-vm:547]: same

File: kvm-install-vm (hunk 1 above)
Comment (ronzle) [reply kvm-install-vm]: fixed

File: kvm-install-vm (hunk 2 above)
Comment (ronzle) [reply kvm-install-vm]: fixed

**Review by giovtorres — APPROVED** [review]
LGTM 👍🏼  Thank you for your contribution

**Pull request merged**